tests/test_input_module.py

```diff
@@ -1,12 +1,13 @@
 """
 Test module for the inputs module of the quicfire_tools package.
 """
-
 from pathlib import Path
 
 import pytest
 from pydantic import ValidationError
 
+import sys
+sys.path.insert(0, "/Users/ntutland/Documents/Projects/quicfire-tools")
```

Comment:
Are you still having issues importing quicfire-tools into tests?

Reply:
Nope  just forgot to take that out